vscode/quint-vscode/scripts/prepare-release.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+QUINT_VERSION=0.5.2
+
```

Comment:
this script depends on being run from a particular directory. instead lets use 

```
# The directory of this file
DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

to get the dir where this script lives and then use that to ensure we are in the appropriate dir.

Reply:
Cool! I used this to `cd` into the parent directory of it, which should always be `quint-vscode` unless the script itself is moved. Is that what you had in mind?

Reply:
Yep! Perfect. Thanks!